internal/models/policy/recipe/network/allow_all.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+Code generated by go-swagger; DO NOT EDIT.
+*/
+
+package policyrecipenetworkmodel
+
+import "github.com/go-openapi/swag"
+
+// VmwareTanzuManageV1alpha1CommonPolicySpecNetworkV1AllowAll allow-all schema
+//
+// swagger:model VmwareTanzuManageV1alpha1CommonPolicySpecNetworkV1AllowAll
+type VmwareTanzuManageV1alpha1CommonPolicySpecNetworkV1AllowAll struct {
```

Comment:
Lets start using smallnames, the model names are getting too long

Reply:
Is `V1alpha1CommonPolicySpecNetworkV1AllowAll ` better?

Reply:
Yes, thats sounds better

Reply:
Done